pkgs/by-name/co/conduwuit/package.nix

```
@@ -84,9 +85,12 @@ rustPlatform.buildRustPackage rec {
 
   passthru = {
     updateScript = nix-update-script { };
-    tests.version = testers.testVersion {
-      package = conduwuit;
-      version = "${version}";
+    tests = {
+      inherit (nixosTests) conduwuit;
+      version = testers.testVersion {
```

```suggestion
    tests = lib.optionalAttrs stdenv.hostPlatform.isLinux {
      inherit (nixosTests) conduwuit;
    } // {
      version = testers.testVersion {
```
This helps avoid OfBorg evaluating and trying to build this on incompatible systems